web/modules/custom/employees/src/Commands/CustomCommands.php

```diff
@@ -124,4 +124,31 @@ public function mark_unused_files_as_temp()
       print("Marked {$file->getFilename()} {$file->id()} as temp\n");
     }
   }
+
+  /**
+   * Drush command to reset the user sync process so the next cron run will restart the sync process.
+   */
+  #[CLI\Command(name: 'employees:reset_user_sync')]
+  #[CLI\Usage(name: 'employees:reset_user_sync', description: 'Reset the user sync process')]
+  public function reset_user_sync()
+  {
+    // Clear all items in the queue
+    /** @var QueueFactory $queue_factory */
+    $queue_factory = \Drupal::service('queue');
+    /** @var QueueInterface $queue */
+    $queue = $queue_factory->get('user_sync');
+    if( $queue != null ) $queue->deleteQueue();
+
+    // Set the Day of Week to today
+    \Drupal::state()->set('epgov.user_sync.day_of_week', date("l"));
```

Comment:
Won't this change all future user sync cron runs from Saturday to whatever day this drush command was run on>

Reply:
Good point. Switched to use a different State variable "sync_now" to override the check on day of the week.